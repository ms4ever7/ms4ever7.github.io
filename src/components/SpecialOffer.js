
import React from 'react';

const SpecialOffer = () => <div className='special-offer__container layout-row layout-align-center-center'>
  <div className='special-offer__title'>Free shopping on orders of 75$+</div>
</div>

export default SpecialOffer;