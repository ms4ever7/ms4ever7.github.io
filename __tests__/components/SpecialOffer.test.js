import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import SpecialOffer from '../../src/components/SpecialOffer';

describe('SpecialOffer', () => {
  let wrapper;


  it('should render correctly', () => {
    wrapper = shallow(
      <SpecialOffer />
    );

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });

  it('has appropriate classname', () => {
    const wrapper = shallow(<SpecialOffer />);

    expect(wrapper.find('.special-offer__title').exists()).toBeTruthy();
  });
});
