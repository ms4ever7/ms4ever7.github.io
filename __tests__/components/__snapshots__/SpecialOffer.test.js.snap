// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExploreStrainsList should render correctly 1`] = `
<div
  className="special-offer__container layout-row layout-align-center-center"
>
  <div
    className="special-offer__title"
  >
    Free shopping on orders of 75$+
  </div>
</div>
`;

exports[`SpecialOffer should render correctly 1`] = `
<div
  className="special-offer__container layout-row layout-align-center-center"
>
  <div
    className="special-offer__title"
  >
    Free shopping on orders of 75$+
  </div>
</div>
`;
